Todo/TaskListsViewController.swift

```diff
@@ -92,7 +92,7 @@ class TaskListsViewController: UITableViewController, UISearchResultsUpdating {
         
         let row = listRows![indexPath.row] as CBLQueryRow
         
-        cell.textLabel?.text = row.key as? String
+        //cell.textLabel?.text = row.key as? String
```

Comment:
row.key is conflicted with row.key(atIndex) method. I think a temp workaround would be 'row.value(forKey: "key")'. Can you try that?